getting-started/events.md

```diff
@@ -0,0 +1,268 @@
+---
+title: "Events"
+weight: 20
+---
+
+# Events
+
+
+### Introduction
+
+Events are portions of data which can be sent to a chat ([send_event](../../agent-api/client-server#send-event)). There are different types of events:
+
+ - [message](#message)
+ - [annotation](#annotation)
+ - [filled-form](#filled-form)
+ - [system message](#system-message)
+ - [file](#file)
+ - [custom event](#custom-event)
+
+
+[Incoming event](../../agent-api/server-client#incoming-event) push will inform you about events sent to a chat (on both, agent and customer side).
+
+
+### Common event fields
+
+All events have some common fields, we will describe them here to avoid duplication:
+
+| name        | puprose                                        | filled by | type   |
+|-------------|------------------------------------------------|-----------|--------|
+| `id`        | id of the event                                | server    | string |
+| `custom_id` | is an ID that you can set for your own purpose | client    | string |
+| `order`     | event with lower `order` are hronologicaly before | server | int    |
+| `author_id` | this is id of sender of the event              | server    | string |
+| `text`      | this is payload that you can send as event (in example message content) | client | string |
+| `recipients` | posibble values are `"all"`(default) and `"agents"` | client | string |
+| `properties` | event properties, more about this in future properties section | client | properties object |
+
+
+### Message
+
+Message is the most common and self-explanatory event type. It allows sending some text to other chat users. The data format looks like this:
```

Comment:
Message is the most common and self-explanatory event type. It allows sending some text to other chat users. The data format looks like this: -> A message is the most common and self-explanatory event type. It allows you to send textual content to other chat users. Its data format looks like this: